specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateEndpointConnection.json

```diff
@@ -46,16 +46,16 @@
         },
         "parameters": [
           {
-            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+            "$ref": "../../stable/2019-08-01/cosmos-db.json#/parameters/subscriptionIdParameter"
```

Comment:
Please don't reference files in a different API version. This makes it extremely difficult to untangle dependencies.